airbyte-integrations/connectors/source-mixpanel/source_mixpanel/manifest.yaml

```diff
@@ -379,15 +485,295 @@ definitions:
               - name
             value: "{{ stream_slice.get('funnel_name') }}"
 
+  # https://developer.mixpanel.com/reference/raw-event-export
+  export_stream:
+    type: DeclarativeStream
+    name: export
+    $parameters:
+      name: export
+    retriever:
+      type: SimpleRetriever
+      requester:
+        $ref: "#/definitions/requester"
+        path: export
+        http_method: GET
+        url_base: "https://data-{{ '' if config.region == 'US' else config.region+'.' }}mixpanel.com/api/2.0/"
+        error_handler:
+          type: CustomErrorHandler
+          class_name: "source_mixpanel.components.ExportErrorHandler"
+          max_retries: 10
+          backoff_strategies:
+            $ref: "#/definitions/default_error_handler/backoff_strategies"
+          response_filters:
+            $ref: "#/definitions/default_error_handler/response_filters"
+      record_selector:
+        type: RecordSelector
+        schema_normalization: Default
+        extractor:
+          class_name: "source_mixpanel.components.ExportDpathExtractor"
+          field_path: []
+    incremental_sync:
+      type: DatetimeBasedCursor
+      cursor_field: time
+      cursor_datetime_formats:
+        - "%Y-%m-%dT%H:%M:%S%z"
+        - "%Y-%m-%d %H:%M:%S"
+        - "%Y-%m-%d"
+      datetime_format: "%Y-%m-%dT%H:%M:%SZ"
+      lookback_window: 'PT{{ max(config.get("attribution_window", 5) * 24 * 60 * 60, config.get("export_lookback_window", 0)) }}S'
+      step: 'P{{ config["date_window_size"] or 30 }}D'
```

Comment:
Wouldn't this generate a key error is `date_window_size` is not defined as part of the config? It seems like with the current code, we would only fallback on 30 if `config["date_window_size"] == 0`

Reply:
updated it